### ISSUE TITLE
Cant download your mod

### ISSUE THREAD

**Issue by NewFrontier15** [justin-pierce/mc-twig-to-stick#1]
I found your mod on the Discord of Minecolonies and since I am using Twigs in my modpack I thought it would fit well, but I can't download your Mod here on Github. Clicking on any of the links on the Code Page opens that specific code. I hope you can fix it 
yours NewFrontier

**Comment by justin-pierce** [comment]
I just created a release to make it easier to download and added installation instructions to the readme. Let me know if that helps. 

If you run into something like this that doesn't have a release in the future, you can also download the latest code of a repo's main branch by clicking the green `Code` button on the repo's main page, then at the bottom of that submenu click on `Download ZIP`. See screenshot: 
![Screen Shot 2024-10-18 at 11 38 13](https://github.com/user-attachments/assets/5e629099-cb58-4ad0-9ccd-d43bbca775c6)

Generally, you'll want to download from Releases though -- it's possible for the main branch of a repo to have unstable code. In this case it's the same (apart from the updated readme).

**Comment by NewFrontier15** [comment]
It worked now thank you

**Comment by NewFrontier15** [comment]
I added the Datapack as described but it shows up as Incompatible (I am running MC 1.20.1 with with Colonies 1.1.689 snapshot)

**Comment by NewFrontier15** [comment]
![2024-10-18_20 04 55](https://github.com/user-attachments/assets/32ae491c-f3ea-4a85-b8ec-3c1c3178cd19)
this is the datapack list

**Comment by NewFrontier15** [comment]
nvm it worked with the 1.0.0. Version

**Comment by justin-pierce** [comment]
So I can try to reproduce the issue, where were you seeing the "incompatible" message?

**Comment by NewFrontier15** [comment]
it was when I was creating a copy of the World with the Data pack. Other mods show up as Incompatible too. And my problem was, that I downloaded the main file first (so the code) clearly that was't working. As soon as I used the 1.0.0 Version it worked I played with it the last hour but here is a screenshot: 
![Bild_2024-10-18_212832793](https://github.com/user-attachments/assets/b1c3a0c0-8624-4300-9a31-ebbaee93ee40)
